lib/gateway/index.js

```diff
@@ -49,11 +44,19 @@ module.exports = function ({ plugins, config } = {}) {
     });
 };
 
-function bootstrap({ plugins, config } = {}) {
-  const app = express();
-  app.set('x-powered-by', false);
+const bootstrapPolicies = ({ app, plugins, config } = {}) => {
+  if (plugins && plugins.policies && plugins.policies.length) {
+    plugins.policies.forEach(policy => {
+      log.debug(`registering policy ${policy.name}`);
```

Comment:
Can we add in this statement that this is coming from a plugin?

Reply:
sure, will do